File: namsor/typescript-node/src/CastegroupIndianFullExample.ts
import * as fs from 'fs';
import api from "namsor_client"
import models from "namsor_client"

const apiCaller = new api.IndianApi();
apiCaller.setApiKey(api.IndianApiApiKeys.api_key, "YOUR_API_KEY");

apiCaller.castegroupIndianFull(
  "IN-UP", // subDivisionIso31662
  "Akash Sharma", // personalNameFull
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling IndianApi#castegroupIndianFull:");
  console.log(error.body);
});

File: namsor/csharp/src/OSEG.NamsorExamples/CastegroupIndianFullExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using App.Namsor.Api;
using App.Namsor.Client;
using App.Namsor.Model;

namespace OSEG.NamsorExamples;

public class CastegroupIndianFullExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey = new Dictionary<string, string> {["api_key"] = "YOUR_API_KEY"};

        try
        {
            var response = new IndianApi(config).CastegroupIndianFull(
                subDivisionIso31662: "IN-UP",
                personalNameFull: "Akash Sharma"
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling IndianApi#CastegroupIndianFull: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
